assessment/services/assessment.py

```diff
@@ -80,6 +80,8 @@ def validate_response_test(request_data):
         raise InvalidResponseTestRegistration('Prompt Should Not Be Empty')
     if len(request_data.get('subject').split()) == 0:
         raise InvalidResponseTestRegistration('Subject Should Not Be Empty')
+    if len(request_data.get('sender').split()) == 0:
```

Comment:
Hello, Arya. May I know why do we need a split in the validation?
Thanks!

Reply:
Hi Jo, if condition be `if request_data.get('sender') is None` wouldn't that allow empty strings too, which wouldn't be allowed in the first place?
Maybe a better suggestion would be just to remove the .split() altogether and check if the string is an empty string or not by `if len(request_data.get('sender')) == 0`. What do you think? 
Thanks!

Reply:
Yes, that would be a correct approach. However, this expression can be further simplified by using `if not request_data.get('sender')` instead.

Reply:
Hi, Jo and Indi.
I tried to detect whitespace input using the split method. If I only check the length of the string, it's still possible for someone to enter '     ' and get a length result that is more than 0. What do you think?
Thanks.

Reply:
Okay, Arya. No problem. As long as the software is working and no hidden bugs are present, everything is okay :)